tests/test_application.py

```diff
@@ -1944,6 +1944,7 @@ async def raise_method(*args, **kwargs):
 
         assert len(recwarn) == 0
 
+    @pytest.mark.flaky(3, 1)  # We could run into a flood error from run_webhook
```

Comment:
shouldn't flood errors xfail thanks to our custom test-request-class?

Reply:
Yes, they do xfail, but loop.call_later will trigger the raise SystemExit, which then fails the test so

Reply:
mh, okay. Then please update the comment to highlight this connection between flood errors and call_later